DirectProgramming/DPC++FPGA/Tutorials/GettingStarted/fpga_template/README.md

```diff
@@ -0,0 +1,92 @@
+
+### On a Windows* System
+This design uses CMake to build the project, and generate build artifacts for `nmake` like this:
+
+```bash
+mkdir build
+cd build
+cmake -G "NMake Makefiles" ..
+```
+
+This project can build 8 targets for Windows.
+
+| Target           | Expected Time  | Output                                                        | Description                                                                                                                                                                                                                                                                                    |
+|------------------|----------------|---------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `nmake fpga_emu` | Seconds        | x86-64 binary                                                 | Compiles the FPGA device code to the CPU. Use the Intel® FPGA Emulation Platform for OpenCL™ software to verify your SYCL code’s functional correctness.                                                                                                                                       |
+| `nmake report`   | Minutes        | RTL                                                           | Compiles the FPGA device code to RTL and generates an optimization report that describes the structures generated on the FPGA, identifies performance bottlenecks, and estimates resource utilization. This report will include the interfaces defined in your selected Board Support Package. |
+| `nmake fpga`     | Multiple Hours | Quartus Place & Route (Full accelerator) + x86-64 host binary | Compiles the FPGA device code to RTL and generate an FPGA image that you can run on a supported accelerator board.                                                                                                                                                                             |
+| `nmake fpga_sim` | Minutes        | RTL + x86-64 binary                                           | Compiles the FPGA device code to RTL and generates a simulation testbench. Use the Questa*-Intel® FPGA Edition simulator to verify your design.                                                                                                                                                |
+
+The `fpga_emu`, `fpga_sim`, and `fpga` targets also produce binaries that you can run. The executables will be called `TARGET_NAME.fpga_emu.exe`, `TARGET_NAME.fpga_sim.exe`, and `TARGET_NAME.fpga.exe`, where `TARGET_NAME` is the value you specify in `src/CMakeLists.txt`.
```

Comment:
I thought you had removed the need for these in your PR we talked about on Wednesday?

Reply:
I got the approval to do so - the new top level readme will be merged this week, and we'll clean up all code samples individual README in a bulk PR after wards to make them all consistent.
So for now it makes sense to have them, so that they are bulk removed/replaced with a pointer to the top level readme afterwards